source/includes/administration/_environments.md

```diff
@@ -273,7 +259,58 @@ Optional | &nbsp;
 `name`<br/>*string* | The updated name of the environment. Should be unique in the environment and only contain lower case characters, numbers, dashes and underscores.
 `description`<br/>*string* | The updated description of the environment
 `membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to  choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*
-`roles`<br/>*Array[[Role](#administration-roles)]* | Update the users roles in the environment. Also, can also update the default role.<br/>*required*: `name`, `users.id`<br/>*optional*: `isDefault`
+
+Response | &nbsp;
```

Comment:
Should we show/tell that the response will be a TaskId and we must use it to get the below response?
Or is this what's returned immediate;y?

Reply:
this is what is returned immediately: 
```
{
  data: ...,
  taskId: ...,
  taskStatus: ...,
}
```

Reply:
Ohh cool. Didn't see the task info at the end.